README.md

```diff
@@ -1,5 +1,6 @@
 # Status - a Mobile Ethereum Operating System
 
+[<img src="https://help.opportunitynetwork.com/hs-fs/hubfs/apple.png" alt="Get it on Apple App Store" height="82"/>](https://apps.apple.com/us/app/status-gateway-to-ethereum/id1178893006)
```

Comment:
What is this domain `opportunitynetwork.com`? Like the other store links, it's better if we refer to the `apple.com` domain. Example here https://developer.apple.com/app-store/marketing/guidelines/